docs/interfaces/commands-and-shells.rst

```diff
@@ -107,6 +107,17 @@ Each noun also provides a ``help`` verb that describes the possible verbs for th
 provide the ``-h`` or ``--help`` argument anywhere, which causes the CLI to exit after printing a help
 message for the object or action specified to that point.
 
+Environment Variables
+=====================
+
+-  ``DET_MASTER``: The network address of the master of the Determined installation. The value can
+   be overridden using the ``-m`` flag.
+-  ``DET_USERNAME``: Specifies the current Determined user. The value can be overridden using the ``-u``
+   flag.
+-  ``DET_PASSWORD``: Specifies the password of the current Determined user. This value can be overriden
+   by running ``det login <username>`` then entering the different password in the prompt.
```

Comment:
Can this be rephrased to make it clearer that `det user login` is the preferred mechanism, but DET_USERNAME and DET_PASSWORD can be used in a pinch where non-interactive behavior is required?